src/ebanx.js

```diff
@@ -70,9 +70,9 @@ EBANX.errors = (function () {
       'pt_BR': {
```

Comment:
@gpressutto5 @andercampanha @RafaelBruno what you think to init separate files? This looks like a good path to start

Reply:
👍

Reply:
👍

Reply:
Good Ideia, let's start the organization for here :)